src/modules/SearchPage/components/MoveCard/styled.ts

```diff
@@ -1,11 +1,13 @@
 import styled from 'styled-components';
 
-export const Card = styled.div`
+export const Card = styled.button`
```

Comment:
I think button should not be called Card =)

Reply:
Actually I changed it back to div. Later on I will read about it (accesability and stuff).